revolv/project/models.py

```diff
@@ -219,20 +219,29 @@ class Project(models.Model):
         null=True,
         help_text='When was the organization being helped established?'
     )
+
     org_name = models.CharField(
         'Organization Name',
         max_length=255,
         help_text='What is the name of the organization being helped?'
     )
+
     mission_statement = models.TextField(
         'Organization Mission',
-        help_text='What is the mission statement of the organization being helped by this project?'
+        help_text='What is the mission statement of the organization being helped by this project?',
+        default='This is the mission statement!',
     )
+
     org_about = models.TextField(
         'Organization Description',
         help_text='Elaborate more about the organization, what it does, who it serves, etc.'
     )
 
+    description = models.TextField(
+        'Project description',
+        help_text='Elaborate more about the project, the goals, etc.'
```

Comment:
This isn't really the right help text to put for the description: should be something referencing the fact that this is the actual body of content that will be shown on the project page, not something that the admins will see when the project is proposed